synchronizer/templates/validation.html

```diff
@@ -25,7 +25,11 @@
                     {% for worklog in worklogs %}
                     <tr class="{% if worklog.is_valid %} table-success {% else %} table-danger {% endif %}">
                         <th scope="row">{{loop.index}}</th>
-                        <td>{{worklog.comment}}</td>
+                        <td>{{worklog.comment}}
+                            {% if not worklog.is_valid %}
+                                <div>Error: Issue id is not present in Jira</div>

```

Comment:
This message should also be shown for any other connector. For example, if I want to export my worklogs to Gitlab and in Gitlab there is no issue found.